python/paddle/v2/fluid/profiler.py

```diff
@@ -49,3 +49,48 @@ def cuda_profiler(output_file, output_mode=None, config=None):
     # Disables profiler collection.
     core.nvprof_stop()
     os.remove(config_file)
+
+
+def reset_profiler():
+    core.reset_profiler()
+
+
+@contextmanager
+def profiler(state, sorted_key=None):
+    """The profiler interface.
+    Different from cuda_profiler, this fuction can be used to profile both CPU
+    and GPU program.
+
+    Args:
+        state (string) : The profiler state, It should be 'CPU' or 'GPU'.
```

Comment:
Feel that it is necessary to add more explanation about param `state`. Users may be confused why a state is needed after the place is given.

Reply:
Add more explanation here.